src/Storages/MergeTree/MergeTreeData.cpp

```diff
@@ -762,6 +762,50 @@ void MergeTreeData::MergingParams::check(const StorageInMemoryMetadata & metadat
             throw Exception("Version column " + version_column + " does not exist in table declaration.", ErrorCodes::NO_SUCH_COLUMN_IN_TABLE);
     };
 
+    /// Check that if the sign column is needed, it exists and is of type Int8.
+    auto check_sign_version_column = [this, & columns](bool is_optional, const std::string & storage)
```

Comment:
I see that the method is invoked only once and `is_optional` is  always set to true. Why do we need it then?

Reply:
I set it as the other lambda function `check_sign_column` and `check_version_column`, in case of (later) someone needs it. 
It's also to keep coherency in the way these function work and are called.